test/extensions_test.dart

```diff
@@ -1352,6 +1353,122 @@ void main() {
         expect(l3.toList(), [4, 5]);
       });
     });
+    group('get frequencies tests', () {
```

Comment:
Just `'frequenices'`, it's cleaner.

The names of groups and tests are concatenated with a space between them, so they can rely on context. Things like "should be correct" is a given for tests, so it doesn't add value. Consider:
```dart
    group('frequencies', () { 
      test('of integers', () { ... });
      test('of strings', () { ... });
      test('of empty iterable', () { ... });
      test('of empty single element iterable', () { ... });
      group('where equality is not identity,', () {
        test('of records', () { ... });
        /// ...
      });
   });
```
These names are only important when a test fails.
At that point, they should give enough context to make the error message meaningful.

Take "frequencies of single element iterable" with an error (if the result was empty) of 
```
  Expected: {42: 1}
    Actual: {}
     Which: has different length and is missing map key <42>
```
With that headline, it's fairly clear where and what the problem is.